lte/gateway/c/session_manager/OperationalStatesHandler.h

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <folly/dynamic.h>
+#include <folly/json.h>
+#include <google/protobuf/util/json_util.h>
+
+#include "SessionStore.h"
+
+namespace magma {
+const std::string TYPE                  = "type";
+const std::string SUBSCRIBER_STATE_TYPE = "subscriber_state";
+const std::string DEVICE_ID             = "device_id";
+const std::string VALUE                 = "value";
+const std::string SESSION_ID            = "session_id";
+const std::string IMSI                  = "imsi";
+const std::string MSISDN                = "msisdn";
+const std::string IPV4                  = "ipv4";
+const std::string ACTIVE_POLICY_RULES   = "active_policy_rules";
+
+std::list<std::map<std::string, std::string>> get_operational_states(
```

Comment:
I have been working on session serialization this weekend, and I think in the near future we should use `serialize_stored_session` and send it. Otherwise, this is yet another way to represent a session that we will have to maintain. 

Unfortunately this is easy to say it but not easy to do: the `serialize_stored_session` is not defined properly and it has lots of nested `toJson` which creates a json object inside a json object inside a json object. So we should fix that first.

Reply:
Good catch, I completely forgot we have the memory store version. Let me double check.

Reply:
@uri200 now in event base loop